### DISCUSSION TITLE
好像没法精简配置了？

### DISCUSSION THREAD

**Discussion by Luoooio** [v2fly/v2ray-core#2172] [Q&A] [answered]
看网上的教程都是比较早的版本，都是修改all.go 文件，然后去实验了一下发现有个包`github.com/v2fly/v2ray-core/v5/main/formats`标注的意思是不能选择性编译(This disable selective compile)。但一但使用了这个包就会直接增加6m左右，太大了啊 ！。！
然后发现这个包的作用是将config转成core.config，要把json配置转成pb好像必须要这个包？有没有办法不用这个包，我就想把json配置硬编码进去。
实在没找到解决方法，感觉无解

**Comment by AkinoKaede** [comment, marked as answer]
You can remove it and use JsonV5.
